sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc

```diff
@@ -362,15 +468,74 @@ provides a description of the error.
 Instead, the `save_log` property provides information about a build operation
 that succeeds.
 This might include warning messages or other diagnostics.
-Each source language specification can define specific information that is
-provided in the log.
+All source languages support the `save_log` property, but each source language
+defines the specific information that is provided in the log.
+The `source_language::sycl` language does not define any specific information
+that is provided in the log, so implementations are free to provide any
+information they choose here.
 In general, the log information is intended for human consumption, and the
 format may not be stable across implementations of this extension.
 
 _Effects (1):_ Constructs a `save_log` property with a pointer to a `std::string`.
```

Comment:
Yes, this could be tightened.  See cd3657458ad9bd841da9477b552072bffd27905c.

Reply:
Perfect. Thanks!

Reply:
Just one question: why would we want to allow a `nullptr` being passed there? Wouldn't that make `save_log` a NOOP? Maybe passing `std::string &` makes more sense.

Reply:
I was thinking that it might be useful to allow code like this:

```
auto props = properties{
  //...
  save_log{(enable_log) ? &log : nullptr},
  //...
};
```

Reply:
Perfect. Oversaw  that. Resolving this now :+1: